src/js/components/sharedComponents/sidebar/Sidebar.jsx

```diff
@@ -36,21 +35,33 @@ const Sidebar = ({
     selectedFy,
     pickedYear,
     detectActiveSection = false,
-    fixedStickyBreakpoint = null
+    fixedStickyBreakpoint = null,
+    isGoingToBeSticky = false
 }) => {
     // yPosition, in px, of sections referenced in sidebar
+    const outerReferenceDiv = useRef();
+    const referenceDiv = useRef();
+    const div = useRef();
     const [sectionPositions, setSectionPositions] = useState([]);
     const [sidebarWidth, setSidebarWidth] = useState("auto");
     const [isSidebarSticky, , , handleScroll] = useDynamicStickyClass(referenceDiv, fixedStickyBreakpoint);
```

Comment:
Can we add a comment about the difference between isSidebarSticky and isGoingToBeSticky?